content/agent/faq/network-time-protocol-ntp-offset-issues.md

```diff
@@ -31,7 +31,7 @@ Any significant offset can have undesired effects, and we recommend keeping host
 
 Instructions for syncing the system clock with NTP vary based on the operating system being used. Here are some helpful links for different operating systems.
 
-* **Windows**: [How To Synchronize Microsoft Windows to a NTP Server](http://www.timetoolsglobal.com/2013/06/21/how-to-synchronize-microsoft-windows-to-a-ntp-server-1/)
+* **Windows**: [How To Synchronize Microsoft Windows to a NTP Server](https://support.microsoft.com/en-us/help/816042/how-to-configure-an-authoritative-time-server-in-windows-server)
```

Comment:
should this be "to an NTP Server" or "with an NTP Server"?

Reply:
I'm buying the `with`